mne/commands/tests/test_commands.py

```diff
@@ -242,4 +243,8 @@ def test_flash_bem():
     os.chdir(currdir)
 
 
+def test_show_info():
+    """Test mne show_info"""
+    check_usage(mne_show_info)
```

Comment:
`show_info` is fast enough that you can actually check that it runs on some files. Don't get lazy on us :)